GeekyPS/SpotiStats_flutter
# added spotify sdk and music player interface

Hey @GeekyPS 
Ref - #4 
I have added the spotify SDK and a music player interface to play any given song. Right now, I have passed the song URI statically since there is no URI in the dummy data and this can be changed if the dummy data is modified to have the URI of the song. 
Steps to test the functionality:
1. Go to TODO in music_player_screen.dart and complete it
-Register your app in the [spotify developer portal](https://developer.spotify.com/dashboard/). You also need to create a sha-1 fingerprint and add this and your package name to the app settings on the dashboard as well as a redirect url.
2. Download Spotify App on the device you are testing this
3. Go to any song item and click it as shown in the attached video (Shape of You song will be played sine its hardcoded as mentioned above)


https://user-images.githubusercontent.com/41890434/206621909-f8b761f5-dd1b-43aa-a9f4-20386fcee2f2.mov

Further Steps:
1. Merge this PR if the feature works as expected
2. Raise an issue to add song URI to Songs Dummy Data
3. Once the dummy data is ready, I'll work on playing the dynamically passed song URI

Thanks
